woocommerce/woocommerce
# add: update onboarding task list copies and illustrations

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Updated copies and illustrations according to latest designs: Y5pUYSJPsGEud1vknUZhi8-fi-3352%3A22233



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure that customise-store feature flag is enabled
2. Go through the onboarding task list and check that the copies and illustration match designs

Variations:

- Shipping task only shows up for countries that don't have shipping smart defaults, e.g Afghanistan
- WCPay only shows up in place of the regular payments task if WCPay is installed and activated


<img width="789" alt="image" src="https://github.com/woocommerce/woocommerce/assets/27843274/480335fd-dbcf-4c0f-8848-0761e6aa912e">
<img width="783" alt="image" src="https://github.com/woocommerce/woocommerce/assets/27843274/9014c420-ae61-4401-98d2-5f834c75d22a">
<img width="785" alt="image" src="https://github.com/woocommerce/woocommerce/assets/27843274/99af5b8d-68d2-45af-97b0-6fdde8628c02">
<img width="792" alt="image" src="https://github.com/woocommerce/woocommerce/assets/27843274/c27139d2-8005-4f6a-917a-8e3543ef2736">
<img width="783" alt="image" src="https://github.com/woocommerce/woocommerce/assets/27843274/4050f956-f328-46ef-a0b6-ba2ad7bd1623">
<img width="791" alt="image" src="https://github.com/woocommerce/woocommerce/assets/27843274/8386c793-491e-4ed2-96f2-1fb0cfb3fe05">




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
